ansible_collections/arista/avd/roles/eos_validate_state/tasks/interface_state.yml

```diff
@@ -27,10 +35,18 @@
 
 - name: Validate Port-Channel interfaces state
   assert:
-    that:
-      - interfaces_state.stdout[0].interfaceDescriptions[port_channel_interface.key].interfaceStatus == 'up'
-      - interfaces_state.stdout[0].interfaceDescriptions[port_channel_interface.key].lineProtocolStatus == 'up'
-    fail_msg: "interface status: {{ interfaces_state.stdout[0].interfaceDescriptions[port_channel_interface.key].interfaceStatus }} - line protocol status: {{ interfaces_state.stdout[0].interfaceDescriptions[port_channel_interface.key].lineProtocolStatus }}"
+    that: |
+      ( interfaces_state.stdout[0].interfaceDescriptions[port_channel_interface.key].interfaceStatus == 'up' and
+      interfaces_state.stdout[0].interfaceDescriptions[port_channel_interface.key].lineProtocolStatus == 'up' and
+      not port_channel_interfaces[port_channel_interface.key].shutdown
+      ) or
+      ( interfaces_state.stdout[0].interfaceDescriptions[port_channel_interface.key].interfaceStatus == 'adminDown' and
+      interfaces_state.stdout[0].interfaceDescriptions[port_channel_interface.key].lineProtocolStatus == 'down' and
+      port_channel_interfaces[port_channel_interface.key].shutdown
+      )
+    fail_msg: "interface shutdown: {{ port_channel_interfaces[port_channel_interface.key].shutdown }}
```

Comment:
you are combining a string and a list into the same variable. I doubt this will work as intended.